version

```diff
@@ -1 +1 @@
-0.9.12
+0.9.14
```

Comment:
The previous version was 0.9.12, should this be 13?

Reply:
@banshee I think we have not made the previous deployment with fix update (was deployed to testnet, but not mainnet) and in that fix the version was changed from 0.9.12 to 0.9.13
https://github.com/Sifchain/sifnode/pull/2120
I think we should merge that PR first (also fixes integration tests), and then do this one. Does it sound good?
cc @intl-man @AustinoBombino @Jedi2002

Reply:
@iverc Agreed. #2120 has been merged.